tests/testthat/test-linetype.R

```diff
@@ -15,11 +16,10 @@ dasharrayPattern <-
          ";")
 
 rect.xpaths <- 
-  c('//svg[@id="numeric"]//rect',
-    '//svg[@id="character"]//rect',
+  c('//svg[@id="numeric"]//g[@class="PANEL1"]//rect',
+    '//svg[@id="character"]//g[@class="PANEL1"]//rect',
     '//td[@id="numeric_legend"]//rect',
```

Comment:
Hey Kevin thanks for the nice description of your modifications. Yes it does make sense to change this test. Next time please consider using an inline comment like this so we can also see the diff.